checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -108,15 +113,15 @@
     protected final Set<Class<? extends Annotation>> initAnnos;
 
     /**
-     * String representation of all initialization annotations.
+     * CanonicalName String representation of all initialization annotations.
      *
-     * <p>{@link UnknownInitialization} {@link UnderInitialization} {@link Initialized} {@link
-     * FBCBottom}
+     * <p>{@link UnknownInitialization}, {@link UnderInitialization}, {@link Initialized}, {@link
```

Comment:
Would this look nicer in javadoc if it's simply `@see` tags, at the end of the javadoc block instead of this odd listing here?

Reply:
Done.